src/odemis/gui/conf/data.py

```diff
@@ -766,6 +766,28 @@
                 "label": "Filter",
             }),
         )),
+    stream.ScannedTCSettingsStream:
+        OrderedDict((
+            ("dwellTime", {
+                "control_type": odemis.gui.CONTROL_SLIDER,
+                "scale": "log",
+            }),
+            ("repetition", {
+                "control_type": odemis.gui.CONTROL_COMBO,
+                "choices": util.resolution_from_range_plus_point,
+                "accuracy": None,  # never simplify the numbers
+            }),
+            ("pixelSize", {
+                "control_type": odemis.gui.CONTROL_FLT,
+            }),
+            ("power", {
```

Comment:
power and period should be just after the dwellTime (as it's the convention to put the detector settings, then emitter, and then extra acquisition settings). However, once you manage to use power and period as "local settings" (ie, with the hwvas argument on the stream), the config will be picked up from "light". In this case, you'll need to drop power (which already exists), and move `period` there.